plugins/outputs/http/README.md

```diff
@@ -0,0 +1,36 @@
+# HTTP Output Plugin
+
+This plugin writes to a HTTP Server using the `POST Method`
+
+It requires a `url` name.
+
+### Configuration:
+
+```toml
+# Send telegraf metrics to HTTP Server(s)
+[[outputs.graylog]]
+  ## Will be transmitted telegraf metrics to the HTTP Server using the below URL.
+  url = "http://127.0.0.1:8080/metric"
```

Comment:
please make it support https then, you can see how output plugins should handle SSL in the influxdb output plugin